src/ff16w_strategy.cpp

```diff
@@ -31,6 +31,58 @@ double FF16w_Strategy::net_mass_production_dt(const FF16_Environment& environmen
   return net_mass_production_dt_A(assimilation_, respiration_, turnover_);
 }
 
+double FF16w_Strategy::evapotranspiration_dt(double area_leaf) const {
+  return 2 * area_leaf;
```

Comment:
@itowers1 - this is the interesting part of the model, but is currently doing very little.